index.bs

```diff
@@ -6384,9 +6383,8 @@ more of the following are redefined in accordance with the rules for exotic obje
 \[[Call]],
 \[[Set]],
 \[[DefineOwnProperty]],
-\[[GetOwnProperty]],
-\[[Delete]] and
-\[[HasInstance]].
```

Comment:
We're still [modifying [[HasInstance]]](https://heycam.github.io/webidl/#es-legacy-callback-interface-object-hasinstance) for legacy callback interfaces, so you might want to keep it listed here.